RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -2292,7 +2292,7 @@ boolean L11_palindome()
 			auto_log_info("Attemping to use Map the Monsters to olfact a Bob Racecar.");
 		}
 		boolean advSpent = autoAdv($location[Inside the Palindome]);
-		if($location[Inside the Palindome].turns_spent > 30 && !in_pokefam() && !in_koe() && !auto_is_valid($item[Disposable Instant Camera]))
+		if($location[Inside the Palindome].turns_spent > 30 && !in_pokefam() && !in_koe() && auto_is_valid($item[Disposable Instant Camera]))
```

Comment:
I added `!auto_is_valid($item[Disposable Instant Camera])` in a recent bees hate you fix and made a mistake. there should not be a ! here. in both bees hate you and glover where disposable instand camera cannot be used we do NOT want to abort after 30 adv because they need more.
in paths where the camera is available we DO want to abort after 30 adv spent as it should not take 30 turns if we have the camera